accessibility/index.html

```diff
@@ -0,0 +1,758 @@
+						<p class="fragment">Tun on VoiceOver: command + F5</p>
+                        <p class="fragment">Use the VO Keys for many tasks = control + option</p>
+                    </section>
+                    <aside class="notes">
+				    Instructions for using VoiceOver on an OSX device once it has been enabled. - Demo and Group Activity
+                    </aside>
+				</section>
+                
+                <!-- USING VOICEOVER FOR OSX END -->
+                <section>
+                    <h2>Screen Readers for <span class="gdi">Windows</span></h2>
+                        <ul>
+                            <li><a href="http://www.freedomscientific.com/Products/Blindness/JAWS" target="_blank">JAWS</a></li>  
+                            <li><a href="http://www.nvaccess.org/about/our-story/" target="_blank">NVDA</a> <small>(Open source)</small></li>
+                            <li><a href="http://www.gwmicro.com/Window-Eyes/" target="_blank">Window-Eyes</a></li> 
+                            <li><a href="#" target="_blank">ChromeVox</a> <small>(Chome Plug-in)</small></li>
+                        </ul>
+                </section>
+                        
+                <!-- HOW TO ACTIVATE VOICEOVER FOR OSX END -->
+                        
+				<!-- IMPORTANCE OF SEMANTIC HTML START -->
+				
+				<section>
+				    <h2>Importance of</br><span class="gdi">Semantic HTML Elements</span></h2>
+                    <small>The accessibility API is part of a device's operating system and represents objects in a user interface, exposing information about each object within the application or Web page.</small>
+
+				    <p>Semantic HTML elements are immediately identified by the accessibility API and interpreted by screen readers. Non-semantic elements convey no information to the accessibility API.</p>
+                </section>
+				
+				<!-- IMPORTANCE OF SEMANTIC HTML END -->
+                        
+                <!-- EXAMPLES OF SEMATIC VS NON-SEMANTIC HTML START -->  
+                    
+				<section>
+                    <h2><span class="gdi">Semantic</span> Elements</h2>
+                    <pre><code data-trim contenteditable> 
+                    
+<!-- Link Example -->
+ <a href="http://www.someurl.com">Go to some Web page</a><br/><br/>
+
+ <!-- Button Example -->
+ <button>Click me</button>
+ 
+                    </code></pre>
+                    <a href="#" style="color:blue;font-size:14pt">Go to some Web page</a><br/><button style="width:150px;height:40px;font-size:14pt">Click me</button> 
+				</section>
+                
+                <section>
+                    <h2><span class="gdi">Non-Semantic</span> Elements</h2>
+                    <small>Divs styled with CSS to look semantic</small>
+                    <pre><code data-trim contenteditable>    
+                    
+ <!-- Link Example -->
+ <div class="link-class">Go to some Web page</div>
```

Comment:
Might be useful to say that they might use JavaScript to animate this. What are common examples of this mistake? Navigation in a header or footer, as well as icon links